.eslintrc.js

```diff
@@ -81,6 +81,8 @@ module.exports = {
 				'mocha/no-top-level-hooks': 'error',
 				'mocha/max-top-level-suites': [ 'error', { limit: 1 } ],
 				'no-console': 'off',
+				'no-undef': 'off',
```

Comment:
Needed to avoid eslint complaining about objects in the `global` scope that are indeed defined, just not in the test file's scope (eg. `page`).